example/37_batched_gemm_bias_gelu_gemm_bias/batched_gemm_bias_gelu_gemm_bias_xdl_fp16.cpp

```diff
@@ -0,0 +1,508 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+/*
+Gemm + Gemm fused operation. Computes C_m_o = A_m_k * B0_k_n * B1_n_o
+                                              |------------|
+                                                   Gemm0
+                                              |---------------------|
+                                                       Gemm1
```

Comment:
Update comment string to reflect the purpose of this example

Reply:
done

Reply:
done